docs/blueprints/EP036.rst

```diff
@@ -199,12 +264,11 @@ Enabling pipeline
 Im simple terms, enabling a pipeline will mean to modify every flow so it complies with the pipeline configuration. The process to successfully enable a pipeline is as follows:
 
   1. **of_multi_table** publishes `kytos/of_multi_table.enable_table` event with pipeline content.
-  2. Wait for all NApp to publish their own events with their respective table settings.

```

Comment:
Alternatively, `of_multi_table` can also have a sanity check, like a consistency check, where overtime it could be checking if table groups are in the tables they're supposed to be. In this case we could go without the event confirmation, because we'd have the guarantee that eventually even left over in-flight flows would be moved too, but this could impact important network traffic, so it's not really like a full replacement. So, probably not worth going for in this iteration. One day having consistency might be worth too but low priority for a future release I'd say.

Reply:
Oh, got it. I am adding it back. I am a little concern about the non-installed NApps (`coloring`, `telemetry_int`). We could make so that `of_multi_table` starts the "pipelining" only for the NApps that have answered or wait for all the NApp to answer but `of_multi_table` will need to know which NApp is installed.

Reply:
Right. Yes. You can rely on `controller` to confirm which ones are enabled or have an event for it too. It's up to you to decide on which one you'd go for.

Reply:
Feel free to close this thread when you add it back then. Thanks.